src/NodeChanger/MethodNameChanger.php

```diff
@@ -10,12 +10,29 @@
 {
     /**
      * @param string[] $renameMethodMap
+     * @param string[]|string[] $acceptedAssertionMethods
      */
-    public function renameNode(MethodCall $node, array $renameMethodMap): ?Node
-    {
+    public function renameNode(
+        MethodCall $node,
+        array $renameMethodMap,
+        array $acceptedAssertionMethods = [],
+        ?string $activeFuncCallName = null
+    ): ?Node {
```

Comment:
I guess it should allways return `null`, since it's object and the value is changed.

Use only `return;` for early return statement